src/integrations/restifyApollo.test.ts

```diff
@@ -1,5 +1,5 @@
 import * as restify from 'restify';
```

Comment:
Doing this makes sure that the interfaces are properly exports as opposed to referencing them directly, so tests a few things in one :)